api_tests/package.json

```
@@ -4,10 +4,10 @@
     "description": "COMIT node end-to-end testing suite",
     "main": "index.js",
     "scripts": {
-        "test": "mocha --bail"
+        "test": "harness.js"
     },
     "engines": {
-        "node": "11.10"
+        "node": ">=10.14 <11"
```

:wrench: 
This should do the same thing:
```suggestion
        "node": "^10.14"
```

---

Good to know!

---

FYI: https://jubianchi.github.io/semver-check/#/^11.14/11.30.0